CommonLibSF/src/REL/ID.cpp

```diff
@@ -257,7 +254,7 @@ namespace REL
 					"Current vendor module: {}"sv,
```

Comment:
Might as well reword this to say "Current platform" to match what the paragraph above says.